config/deploy/deployment.yaml.tpl

```diff
@@ -65,7 +65,7 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 100Mi
+            memory: 400Mi
           requests:
             cpu: 100m
-            memory: 100Mi
+            memory: 300Mi
```

Comment:
this is not used in production, but I'm reconciling it agains the values we use in prod.